android/src/main/kotlin/com/adyen/checkout/flutter/components/ComponentLoadingBottomSheet.kt

```diff
@@ -0,0 +1,47 @@
+package com.adyen.checkout.flutter.components
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.FragmentManager
+import com.adyen.checkout.components.core.internal.Component
+import com.adyen.checkout.flutter.R
+import com.adyen.checkout.ui.core.AdyenComponentView
+import com.adyen.checkout.ui.core.internal.ui.ViewableComponent
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+class ComponentLoadingBottomSheet<T>(private val component: T) :
+    BottomSheetDialogFragment() where T : ViewableComponent, T : Component {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.component_bottom_sheet_content, container, false)
+
+    override fun onViewCreated(
+        view: View,
+        savedInstanceState: Bundle?
+    ) {
+        view.findViewById<AdyenComponentView>(R.id.adyen_component_view)?.attach(component, viewLifecycleOwner)
+    }
+
+    companion object {
+        private const val TAG = "AdyenComponentLoadingBottomSheet"
+
+        fun <T> show(
+            fragmentManager: FragmentManager,
+            component: T
+        ) where T : ViewableComponent, T : Component {
+            ComponentLoadingBottomSheet(component).apply {
+                isCancelable = false
```

Comment:
I think it's better to set this in `onCreateView`. Just to make sure it's always set